lib/configproxy.js

```diff
@@ -337,6 +340,7 @@ ConfigurableProxy.prototype.handle_proxy = function (kind, req, res) {
         this.handle_proxy_error(404, kind, req, res);
         return;
     }
+    this.emit("proxy_request", req, res);
```

Comment:
Nah, more like warning signs went off in my head if I was to use this. Not real sure how to document it either.